### PR TITLE
Bar 87 script for retrieving and testing on a trained model

### PR DESCRIPTION
**Pull request by satyarth**

### CODE REVIEW

File: src/local/test.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    loaded = mlflow.pytorch.load_model(f"runs:/{mlflow_run_id}/lassonet-model")
+    loaded.eval()
+    
+    print("Predicting")
+    
+    trainer = Trainer(gpus=0)
+    train_preds, val_preds, test_preds = trainer.predict(loaded, experiment.data_module)
+
+    best_col = int(run.data.metrics['best_alpha_index'])
+    best_val_r2 = r2_score(experiment.y_val, torch.cat(val_preds)[:, best_col].numpy())
```
Comment (dkolobok) [inline src/local/test.py:41]: in the future we will need the metric function and name here and below to be parameters and not hardcoded. you can fix it now or when you deal with non-continuous phenotypes, up to you. I think in other parts of the code I have function arguments of metric_fun=r2_score and metric_name='r2'. This is what I used for TG (categorical output)

**Review by dkolobok — APPROVED** [review]
